static/js/pages/individual-contributions.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var $ = require('jquery');
+
+var tables = require('../modules/tables');
+var helpers = require('../modules/helpers');
+var columns = require('../modules/columns');
+
+var donationTemplate = require('../../templates/receipts.hbs');
+
+$(document).ready(function() {
```

Comment:
Ah, that makes sense. I'm all for changing it, but can we do it in a subsequent PR, since this branch didn't introduce anything new in this regard?